api/src/main/java/org/pmiops/workbench/config/CommonConfig.java

```diff
@@ -18,14 +19,13 @@ JsonFactory jsonFactory() {
     return new JacksonFactory();
   }
 
-  @Bean
```

Comment:
Removing HttpTransport from CommonConfig allows us to make CommonConfig actually be shared across our three main configs: WebMvc, CommandLineTool, and IntegrationTest.